hexrd/ui/main_window.py

```diff
@@ -878,17 +878,22 @@ def view_fit_grains_config(self):
 
     def new_mouse_position(self, info):
         labels = []
-        labels.append('x = {:8.3f}'.format(info['x_data']))
-        labels.append('y = {:8.3f}'.format(info['y_data']))
+        labels.append(f'x = {info["x_data"]:8.3f}')
+        labels.append(f'y = {info["y_data"]:8.3f}')
         delimiter = ',  '
 
         intensity = info['intensity']
         if intensity is not None:
-            labels.append('value = {:8.3f}'.format(info['intensity']))
-            labels.append('tth = {:8.3f}'.format(info['tth']))
-            labels.append('eta = {:8.3f}'.format(info['eta']))
-            labels.append('dsp = {:8.3f}'.format(info['dsp']))
-            labels.append('hkl = ' + info['hkl'])
+            if isinstance(intensity, np.ma.MaskedArray):
+                masked_label = '--'
```

Comment:
@joelvbernier If you'd rather this be something other than `--`, let me know. Matplotlib masks both `nan` and `inf`, so it could be either.

If it's preferable, we could also try to grab the unmasked value [here](https://github.com/HEXRD/hexrdgui/blob/36831d1fc8360ee26dae12b6cd8199b0356817f2/hexrd/ui/image_tab_widget.py#L266), and then display what it actually is, either `nan` or `inf`.

Reply:
`nan` is probably the best.

Reply:
LGTM

Reply:
Okay, it's fixed. It now actually displays whatever the underlying data is: `nan` or `inf`.